launch/launch.go

```diff
@@ -0,0 +1,105 @@
+package launch
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/screwdriver-cd/sd-local/config"
+	"github.com/screwdriver-cd/sd-local/screwdriver"
+)
+
+var lookPath = exec.LookPath
+
+// Runner prepares binaries to run builds and run the specified build.
+type Runner interface {
+	RunBuild(buildConfig buildConfig) error
+	SetupBin() error
+}
```

Comment:
Do we need to export `RunBuild` `SetupBin` and `Runner` ?

Reply:
fixed